.env.example

```diff
@@ -3,7 +3,9 @@ ENVIRONMENT="staging"
 
 # URLs & Paths
 DEFAULT_SITE_URL=""
+# URL that will be prefixed to the asset URL - commonly the root URL
 ASSET_BASE_URL=""
+# Path that will be prefixed to the relative path of the asset volume - commonly the Web directory
```

Comment:
Similar points as above. Additionally:
- This should be the full _absolute_ path, not a relative path. Instructions should now create any ambiguity about that.

Reply:
Please see if it looks better now.